blockly-core/core/ui/css.js

```diff
@@ -132,7 +95,10 @@ Blockly.Css.setCursor = function(cursor, opt_svg) {
  */
 Blockly.Css.CONTENT = [
   '.blocklyDraggable {',
-    // Placeholder for cursor rule. Must be first rule (index 0).
+  '  cursor: url(/blockly/media/handopen.cur) 8 5, auto;',
+  '}',
+  '.dragging, .dragging .blocklyDraggable {',
```

Comment:
It's not clear why the cursor needs to be updated in those cases.  I think there was an edge case in the old code where the `.blocklyDraggable` CSS rule needed to be reset back to `OPEN`, but the new approach doesn't have that issue.  When the drag is terminated and the class is removed, all children inherit the correct cursor.